pennylane/interfaces/jax_jit_tuple.py

```diff
@@ -42,11 +43,11 @@ def _validate_jax_version():
         raise InterfaceUnsupportedError(msg)
 
 
-def _copy_tape(t, a):
+def _copy_tape(t, a, unwrap=True):
     """Copy a given tape with operations and set parameters"""
     tc = t.copy(copy_operations=True)
     tc.set_parameters(a, trainable_only=False)
```

Comment:
Is this just going to have the exact same performance problems we are trying to solve?

Reply:
well, it used to do this `set_parameters` and then _another one_ when calling `tape.Unwrap` on this copied tape, so it's better. But yes, we should try to remove all uses of `tape.set_parameters` (assuming it's used with the ultimate goal of unwrapping). TF has the same problem, but perhaps I should do there what jax is doing, and work on improving this in a subsequent PR.